precompile/allow_list.go

```diff
@@ -59,55 +61,44 @@ func (c *AllowListConfig) Equal(other *AllowListConfig) bool {
 	if other == nil {
 		return false
 	}
-	if len(c.AllowListAdmins) != len(other.AllowListAdmins) {
+	if !checkList(c.AllowListAdmins, other.AllowListAdmins) {
```

Comment:
can we update this to use the syntax `if condition := checkList(...,...); !condition {`